installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc

```diff
@@ -23,6 +23,10 @@ include::modules/understanding-agent-install.adoc[leveloffset=+1]
 
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#installation-requirements-platform-none_preparing-to-install-with-agent-based-installer[Requirements for a cluster using the platform "none" option]
 
+* xref:../installing_bare_metal_ipi/ipi-install-prerequisites.adoc#network-requirements-increase-mtu_ipi-install-prerequisites[Increase the network MTU]
+
+* For more information on adding worker nodes to single-node OpenShift clusters, see xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to sigle-node OpenShift clusters].
```

Comment:
AFAIU, the mentioned procedures for adding a worker to an already existing cluster requires an internet connection, so they are not suitable for disconnected environments. Can you please confirm? In such case it could be useful to add a note for such constraint, since the agent-based installer can also be used to deploy a cluster into a disconnected environment

Reply:
@zniu1011 Can you confirm if I have to add the note?

Reply:
@subhtk We need to add the note. Please see "Requirements for installing single-node OpenShift worker nodes" in https://docs.openshift.com/container-platform/4.12//nodes/nodes/nodes-sno-worker-nodes.html#ai-sno-requirements-for-installing-worker-nodes_add-workers

Reply:
Acknowledged. Let me add the note.

Reply:
Hey @zniu1011, can you provide the LGTM label once.

Reply:
@zniu1011 IINW the mentioned procedure requires to access requires (at least) to access https://console.redhat.com/openshift to work, so the access to a local registry would not seem to be enough.

Reply:
Ok, thanks @andfasano . @subhtk Can we please make the notes like this: "Please note that the access to OpenShift Cluster Manager or Assisted Installer API is required using the way to add the worker nodes, so it is not suitable for agent-based installed clusters in disconnected environments."

Reply:
Sure. Let me change it right away.

Reply:
@zniu1011 Done, Please take a look.

Reply:
As long as it sounds reasonable to me, I'd like if someone from the assisted team could double-check the phrasing to ensure it matches properly the mentioned case cc @oourfali